docs/2.8.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -194,6 +194,90 @@ The ping command creates two contracts between the admin parties, then exercises
 Version Specific Notes
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Upgrade to Release 2.8
+^^^^^^^^^^^^^^^^^^^^^^
+
+Version 2.8 extends the database schema. If you use the "migrate and start" feature, the database schema will be automatically updated.
+Otherwise, perform the manual database migration steps outlined in the :ref:`database migration steps <migrating_the_database>`.
+
+Protocol versions 3 and 4 are deprecated
+""""""""""""""""""""""""""""""""""""""""
+Protocol versions 3 and 4 are now marked as deprecated and will be removed in ``2.9``. Protocol version 5 should be preferred for any new deployment.
+
+Configuration changes
+"""""""""""""""""""""
+
+**KMS wrapper-key configuration value**:
```

Comment:
@mlachkar-da so you can kill this section

Reply:
I understand this is not breaking anymore `*KMS wrapper-key configuration value` but there is a change in the configuration as it's the case for "sql batching param" that has been moved (which is also non breaking)

Reply:
Fair enough, you can indicate the change, thanks!